app/routes.py

```diff
@@ -1,2 +1,188 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, make_response, request, abort
+from datetime import date
+from pathlib import Path
+import os
+from dotenv import load_dotenv
+import requests
+from app.models.goal import Goal
 
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+env_path = Path('.')/ '.env'
+load_dotenv()
+
+
+@tasks_bp.route("", methods=["POST", "GET"])
+def handle_tasks():
+    
+    if request.method == "POST":
+        request_body = request.get_json()
+
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return make_response({"details":"Invalid data"}, 400)
+
+        new_task = Task(
+            title=request_body["title"],
+            description=request_body["description"],
+            completed_at=request_body["completed_at"]
+        )
+
+        db.session.add(new_task)
+        db.session.commit()
+
+        return jsonify({"task": new_task.to_dict()}), 201
+
+    elif request.method == "GET":
+        task_response= []
+        if request.args.get('sort') == 'asc':
+            tasks = Task.query.order_by(Task.title.asc()).all()
+        elif request.args.get('sort') == 'desc':
+            tasks = Task.query.order_by(Task.title.desc()).all()
+        else:
+            tasks = Task.query.all()
+
+        for task in tasks:
+            task_response.append(task.to_dict())
+        return jsonify(task_response), 200
+
+@tasks_bp.route("/<task_id>", methods= ["GET", "PUT", "DELETE"])
+def handle_task(task_id):
+    task = Task.query.get(task_id)
+    
+
+    if task is None:
+        return make_response(f"Task {task_id} not found"), 404
```

Comment:
I like how your code is structured so this initial logic can be re-used regardless of whether it's a GET, PUT, or DELETE.